model/cluster_utility.go

```diff
@@ -65,6 +69,19 @@ func NewUtilityMetadata(metadataBytes []byte) (*UtilityMetadata, error) {
 	return &utilityMetadata, nil
 }
 
+// SetUtilityValuesDirectory sets the global location of values &
+// charts for the UtilityGroup. Note: changing this values affects the
+// whole Provisioner, not just a single cluster
+func SetUtilityValuesDirectory(dir string) {
+	utilityValuesDirectory = dir
+}
+
+// UtilityValuesDirectory returns the global location of the values &
+// charts directory
+func UtilityValuesDirectory() string {
+	return utilityValuesDirectory
+}
```

Comment:
I was thinking that instead of returning just the values path the helper function would return `fmt.Sprintf("%s/%s", utilityValuesDirectory, yamlFileName)` or something so the directory is always built the same way.

0/5 on this, but maybe an idea for future improvement.

Reply:
I realize I implemented this a little differently than you originally suggested, so let me explain my thinking a little: I figured that the assembly would be delegated out to the different utilities via `ValuesPath` so that they _don't_ always have to be built the same way. Each Utility then is responsible for tracking its configuration directory and isn't bound to a certain structure beyond the root directory, in the future. Does that make sense @gabrieljackson ?